Emby.Server.Implementations/Networking/NetworkManager.cs

```diff
@@ -95,9 +95,8 @@ private List<IPAddress> GetLocalIpAddressesInternal(bool ignoreVirtualInterface)
 
         private static bool FilterIpAddress(IPAddress address)
         {
-            var addressString = address.ToString();
-
-            if (addressString.StartsWith("169.", StringComparison.OrdinalIgnoreCase))
+            if (address.IsIPv6LinkLocal
+                || address.ToString().StartsWith("169.", StringComparison.OrdinalIgnoreCase))
```

Comment:
Any idea why it's 169?

Reply:
Should also be `169.254.0.0/16`, not `169.0.0.0/8`
https://en.wikipedia.org/wiki/Link-local_address#IPv4

Reply:
@cvium there doesn't seems to be a property/method that can easily replace it (afaik)